### ISSUE TITLE
Thanks to the DJL team

### ISSUE THREAD

**Issue by Jzow** [awslabs/djl#146]
## Question

I have successfully run it on my Mac, 16G of memory on my Mac, I run the TrainMnist.java file,

`[INFO ] - Training on: cpu().
[INFO ] - Load MXNet Engine Version 1.7.0 in 0.140 ms.
Training:    100% |████████████████████████████████████████| Accuracy: 0.93, SoftmaxCrossEntropyLoss: 0.24, speed: 4308.54 items/sec
Validating:  100% |████████████████████████████████████████|
[INFO ] - Epoch 1 finished.
[INFO ] - Train: Accuracy: 0.93, SoftmaxCrossEntropyLoss: 0.24
[INFO ] - Validate: Accuracy: 0.96, SoftmaxCrossEntropyLoss: 0.13
Training:    100% |████████████████████████████████████████| Accuracy: 0.97, SoftmaxCrossEntropyLoss: 0.10, speed: 4234.57 items/sec
Validating:  100% |████████████████████████████████████████|
[INFO ] - Epoch 2 finished.
[INFO ] - Train: Accuracy: 0.97, SoftmaxCrossEntropyLoss: 0.10
[INFO ] - Validate: Accuracy: 0.97, SoftmaxCrossEntropyLoss: 0.10
[INFO ] - train P50: 7.825 ms, P90: 10.198 ms
[INFO ] - forward P50: 0.240 ms, P90: 0.312 ms
[INFO ] - training-metrics P50: 0.011 ms, P90: 0.014 ms
[INFO ] - backward P50: 0.452 ms, P90: 0.567 ms
[INFO ] - step P50: 0.535 ms, P90: 0.704 ms
[INFO ] - epoch P50: 21.252 s, P90: 21.252 s

Process finished with exit code 0`

Train: Accuracy: 0.97,Validate: Accuracy: 0.97

I have a problem，I used  the DeepLearning4j framework before, It's similar to DJL, Can these two be mixed used?
In addition, I would like to use other engines, such as Tensorflow and Pytorch

**Comment by Jzow** [comment]
**Thank you for your prompt reply and technical support ,I am a China Java full stack engineer, I like DJL.**@zachgk @frankfliu @stu1130

**Comment by frankfliu** [comment]
@Jzow 
You can run DL4J and DJL in the same JVM, and you can run MXNet, TensorFlow and PyTorch engine in the same JVM as well. See our multi-engine demo: https://github.com/aws-samples/djl-demo#use-multiple-engines-in-single-jvm

There is an feature request regarding integrate ND4J as one of DJL's engine, and there has some implementation already: https://github.com/awslabs/djl/issues/2

At this moment, we don't have bandwidth to support ND4J yet. If you interested in using ND4J with DJL, you can pick up that work and contribute to DJL.

**Comment by frankfliu** [comment]
@Jzow feel free to reopen this issue if you have further questions.
